skyportal/tests/frontend/test_sources.py

```diff
@@ -33,6 +34,16 @@ def test_comments(driver, user, public_source):
     comment_box.send_keys(comment_text)
     driver.find_element_by_css_selector('[type=submit]').click()
     driver.wait_for_xpath(f'//div[text()="{comment_text}"]')
+    driver.wait_for_xpath('//span[contains(@class,"commentTime")]')
+    timestamp_text = driver.find_element(By.XPATH, 
+                        '//span[contains(@class,"commentTime")]').text
+    day_text, time_text = timestamp_text.split(" at ", 2)
+    timestamp = datetime.strptime(time_text, '%I:%M %p')
+    day = datetime.today()
+    if day_text == 'Yesterday':
+        day = day.AddDays(-1)
+    timestamp = datetime.combine(day, timestamp.time())
+    assert(abs((datetime.now() - timestamp).seconds) < 300)
```

Comment:
Will this ever be longer than a few seconds?  How about setting this to 10 seconds?

Reply:
Okay, I readjusted it to 30 seconds (since it seems to take slightly longer on my VM--which is on my laptop--and I wanted to make sure it would pass)